amplify/backend/function/risestandstrongcb34053a/src/utils/aws-utils.js

```diff
@@ -58,6 +58,44 @@ async function getAnnouncements(){
     }
 }
 
+/**
+ * Put updated announcement into table in DynamoDB.
+ * Returns nothing.
+ */
+async function putAnnouncement(data){
+    const docClient = new AWS.DynamoDB.DocumentClient();
+    var params = {
+        TableName : 'announcementsV3', //current tablename
+        Key: {                         //used to find/not find desired object
+            announcementId: data.announcementId
+        },
+        UpdateExpression: "set title = :title, author = :author, createdAt = :createdAt, link = :link, body = :body", //tells what to update
+        ConditionExpression: "attribute_exists(announcementId)",  //only do the update if the item exits
+        ExpressionAttributeValues:{ //assigns values to updates
+            ":title": data.title,
+            ":author": data.author,
+            ":createdAt": data.createdAt,
+            ":link": data.link,
+            ":body": data.body,
+        }
+    };
+
+    try{
+        let broke = false;  //toggle for if the object is found in the table
```

Comment:
I don't think you need the logic with broke because of your ConditionExpression. It will only update if the announcementId already exists.